pyvrp/Model.py

```diff
@@ -194,8 +185,9 @@ def add_edge(
 
     def add_vehicle_type(
         self,
-        capacity: int,
-        num_available: int,
+        num_available: int = 1,
+        depot: int = 0,
```

Comment:
Need to update this to use previously created `Depot` objects, not leak indices.